dist/platforms/windows/Dockerfile

```diff
@@ -0,0 +1,18 @@
+ARG IMAGE
+FROM $IMAGE
+
+LABEL "com.github.actions.name"="Unity - Builder"
+LABEL "com.github.actions.description"="Build Unity projects for different platforms."
+LABEL "com.github.actions.icon"="box"
+LABEL "com.github.actions.color"="gray-dark"
+
+LABEL "repository"="http://github.com/webbertakken/unity-actions"
```

Comment:
We should probably take this opportunity to change this to `http://github.com/game-ci/unity-actions`